swan-lake/development-tutorials/configurability/provide-values-to-configurable-variables.md

```diff
@@ -93,7 +123,7 @@ $ set BAL_CONFIG_FILES=<path-to-info.toml>;<path-to-Config.toml>
 $ export BAL_CONFIG_FILES=<path-to-info.toml>:<path-to-Config.toml>
 ```
 
-## Provide via environment variables
+### Provide via environment variables with TOML content
```

Comment:
This is under the title with `Provide using TOML content`, Then why do we need to add `with TOML content` to this sub title. IMO this is not necessary and we don't have that part in previous sub title `Provide via configuration files` as well.